jsonp/2.0/_index.md

```diff
@@ -1,27 +1,26 @@
 ---
-title: "Jakarta JSON Processing 2.0 (under development)"
-date: 2020-01-15
+title: "Jakarta JSON Processing 2.0 (milestone)"
+date: 2020-05-31
 summary: "Release for Jakarta EE 9"
 ---
 Jakarta JSON Processing defines a Java(R) based framework for parsing, generating, transforming, and
 querying JSON documents.
 
-* [Jakarta JSON Processing 2.0 Release Record]()
+* [Jakarta JSON Processing 2.0 Release Record](https://projects.eclipse.org/projects/ee4j.jsonp/releases/2.0.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta JSON Processing 2.0 Specification Document]() (PDF)
```

Comment:
You have no plans to have a Spec available for Milestone 1?  That's okay.  But, I wouldn't remove these lines from the _index.md file.  I'd leave them with a blank url as they are.

Reply:
JSONP doesn't have a spec document. Historically, the spec is just javadocs. Javadocs are jakartified. I'll remove these lines.

Reply:
Thanks, @m0mus.  Didn't realize that jsonp relied solely on javadoc.